bloom/release-bloom.py

```diff
@@ -77,6 +79,9 @@ def call_jenkins_jobs(argv):
         params['ROS2'] = True
         ubuntu_per_ros_distro = UBUNTU_DISTROS_IN_ROS2
 
+    if args.ignition_collection:
+        params['IGNITION_COLLECTION'] = args.ignition_collection
```

Comment:
Maybe we should use `IGNITION_VERSION` everywhere to avoid confusion?

Reply:
changed